tests/testthat/test-epinowcast.R

```diff
@@ -115,9 +119,13 @@ test_that("epinowcast() can fit a simple reporting model where the max delay is
   skip_on_local()
 
   pobs_long_delay <- suppressWarnings(
-    pobs$obs[[1]][, .group := NULL] |>
-    enw_filter_reference_dates(include_days = 20) |>
-    enw_preprocess_data(max_delay = 30)
+    enw_preprocess_data(
+      enw_filter_reference_dates(
+        pobs$obs[[1]][, .group := NULL], 
+        include_days = 20
+      ),
+    max_delay = 30
```

Comment:
```suggestion
      max_delay = 30
```
feels like lintr would also indicate on this?